src/aws.rs

```diff
@@ -731,22 +731,23 @@ impl Aws {
         // response parsing from here
         let instances = res
```

Comment:
Can we change the variable name here to `reservations` or `instance_reservations`